scripts/dbx

```diff
@@ -86,21 +90,49 @@ def get_clusters():
     )
     resp.raise_for_status()
 
-    CLUSTERS = {c["cluster_name"]: Cluster(c) for c in resp.json().get("clusters", {})}
+    all_clusters = resp.json().get("clusters", {})
+    echo("Fetched clusters:")
+    echo(
+        "\n".join(
+            [
+                f'{cl["cluster_name"]} | {cl["cluster_id"]} | {cl["state"]}'
+                for cl in all_clusters
+            ]
+        )
+    )
+    # All possible clusters states
+    STATES = [
+        "RUNNING",
+        "PENDING",
+        "RESTARTING",
+        "RESIZING",
+        "TERMINATING",
+        "TERMINATED",
+    ]
+    # There can be more than one cluster with the same name
+    # Sort clusters by state and then cluster ID; if there are multiple clusters with the
+    # same name and different states, this will return the running one first, and a pending
+    # one before a terminated one.  This means that if we retrieved one of two terminated
+    # clusters in order to start one up, when we fetch them again we'll find the one that's
+    # been started up
+    # Note that we need to reverse the sorted list as the CLUSTERS dict will keep the last one
```

Comment:
I had to think hard about this, but now that I have, I think it's a nice alternative to, say, grouping, sorting, and taking the first for each group. Possibly we could save the next person from also thinking hard with

```suggestion
    # We reverse the sorted list, such that RUNNING is last and TERMINATED is first.
    # We then instantiate a Cluster object for each state/name combination,
    # but overwrite earlier instances with later instances, for instances with
    # the same name.
```